wagtailio/project_styleguide/templates/patterns/base_page.html

```diff
@@ -1,138 +1,123 @@
-                    js = d.createElement(s); js.id = id;
-                    js.src = "https://connect.facebook.net/en_US/sdk.js";
-                    fjs.parentNode.insertBefore(js, fjs);
-                }(document, 'script', 'facebook-jssdk'));
-            </script>
-        {% endif %}
-
-        {% if not DEBUG %}
-            <!-- twitter campaign tracking code -->
-            <script src="https://platform.twitter.com/oct.js"></script>
-            <script>
-                twttr.conversion.trackPid('l6ggy', {
-                    tw_sale_amount: 0,
-                    tw_order_quantity: 0
-                });
-            </script>
-            <noscript>
-                <img height="1" width="1" style="display:none;" alt=""
-                    src="https://analytics.twitter.com/i/adsct?txn_id=l6ggy&p_id=Twitter&tw_sale_amount=0&tw_order_quantity=0"
-                />
-                <img height="1" width="1" style="display:none;" alt=""
-                    src="//t.co/i/adsct?txn_id=l6ggy&p_id=Twitter&tw_sale_amount=0&tw_order_quantity=0"
-                />
-            </noscript>
-            <!-- end twitter campaign tracking code -->
-        {% endif %}
-
-        <script src="https://platform.twitter.com/widgets.js" async></script>
-    </body>
-</html>
+{% block title_suffix %} | Wagtail CMS{% endblock %}
+
+{% block meta_tags %}
+    <meta name="description" content="{% firstof page.social_text page.listing_intro %}" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    {% if page.canonical_url %}<link rel="canonical" href="{{ page.canonical_url }}" />{% endif %}
+
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:site" content="@wagtailcms" />
+    <meta name="twitter:creator" content="@wagtailcms" />
+    <meta name="twitter:title" content="{% firstof page.seo_title page.title %} | Wagtail CMS" />
+    <meta name="twitter:description" content="{% if page.social_text %}{{ page.social_text }}{% elif page.listing_intro %}{{ page.listing_intro }}{% elif page.introduction %}{{ page.introduction|truncatechars:160 }}{% endif %}" />
+
+    {% if page.social_image %}
+        {% image page.social_image width-400 as twitter_img %}
+    {% elif page.main_image %}
+        {% image page.main_image width-400 as twitter_img %}
+    {% endif %}
+    {% if twitter_img %}
+        <meta name="twitter:image" content="{{ twitter_img.url }}" />
+        {% if twitter_img.alt %}<meta name="twitter:image:alt" content="{{ twitter_img.alt }}" />{% endif %}
+    {% else %}
+        <meta name="twitter:image" content="{% static 'img/default-sharing-image.png' %}" />
+    {% endif %}
+
+    {% if FB_APP_ID %}<meta property="fb:app_id" content="{{ FB_APP_ID }}" />{% endif %}
+    <meta property="og:type" content="website" />
+    {% wagtail_site as site %}
+    <meta property="og:url" content="https://{{ site.root_url }}{% pageurl page %}" />
```

Comment:
You're using `site` here, does this work as expected with the variable change from `base.html`?

Reply:
it does because we have `{% wagtail_site as site %}`